metricflow/plan_conversion/instance_converters.py

```diff
@@ -558,19 +558,25 @@ def transform(self, instance_set: InstanceSet) -> InstanceSet:  # noqa: D102
         # Sanity check to make sure the specs are in the instance set
 
         if self._include_specs:
+            include_specs_not_found = []
             for include_spec in self._include_specs.all_specs:
                 if include_spec not in instance_set.spec_set.all_specs:
-                    raise RuntimeError(
-                        f"Include spec {include_spec} is not in the spec set {instance_set.spec_set} - "
-                        f"check if this node was constructed correctly."
-                    )
+                    include_specs_not_found.append(include_spec)
+            if include_specs_not_found:
+                raise RuntimeError(
+                    f"Include specs {include_specs_not_found} are not in the spec set {instance_set.spec_set} - "
+                    f"check if this node was constructed correctly."
```

Comment:
nit: this is an odd follow up. If I think it was constructed correctly and I hit this error, what does that indicate?

Reply:
It indicates that you either 1) constructed the instance set correctly or 2) constructed the node incorrectly, since they don't match!

Reply:
This was related to this PR because I ran into this error and that's why I needed to change the instance specs built in the `JoinDescription`. Definitely could have been split into another PR though.